### PR TITLE
upgrade hive

### PR DESCRIPTION
**Pull request by ldpliu**
https://github.com/open-cluster-management/backlog/issues/9948

### CODE REVIEW

**Comment by zhiweiyin318** [comment]
do we also need to update the CRD related with Hive in deploy dir?

File: pkg/webhook/denynamespace/admission_test.go [hunk 1]
```diff
@@ -6,10 +6,15 @@ import (
 	"github.com/stretchr/testify/assert"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
```
Comment (qiujian16) [inline pkg/webhook/denynamespace/admission_test.go:9]: @zhiweiyin318 do we use this admission anywhere?  should we consider removing these codes?

**Pull request merged**